go-controller/pkg/node/gateway_shared_intf.go

```diff
@@ -2026,12 +2029,20 @@ func setNodeMasqueradeIPOnExtBridge(extBridgeName string) error {
 	if config.IPv6Mode {
 		_, masqIPNet, _ := net.ParseCIDR(config.Gateway.V6MasqueradeSubnet)
 		masqIPNet.IP = config.Gateway.MasqueradeIPs.V6HostMasqueradeIP
-		bridgeCIDRs = append(bridgeCIDRs, cidrAndFlags{ipNet: masqIPNet, flags: unix.IFA_F_NODAD})
+		// Depreciate the IPv6 host masquerade IP address to ensure its not used in source address selection except
```

Comment:
@martinkennelly you mean "deprecate" and not "depreciate" in these comments and titles right?